### PR TITLE
Feature/tour page

### PR DESCRIPTION
**Pull request by fominmaksim**
Completed header

### CODE REVIEW

**Review by AndrewKaenko — APPROVED** [review]
Good job. No errors found

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]
again you forgot to remove mocks =) _src/MOCK.jsx_

File: package.json [hunk 1]
```diff
@@ -10,10 +10,12 @@
     "extend-shallow": "^3.0.2",
     "gh-pages": "^3.2.3",
     "react": "^17.0.2",
+    "react-date-picker": "^8.3.5",
```
Comment (alexandrtovmach) [inline package.json:13]: not used ― should be removed

File: package.json [hunk 2]
```diff
@@ -10,10 +10,12 @@
...
     "react-dom": "^17.0.2",
     "react-hook-form": "^7.18.1",
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
+    "react-transition-group": "^4.4.2",
```
Comment (alexandrtovmach) [inline package.json:18]: not used ― should be removed

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**